src/common/MString.cpp

```diff
@@ -175,6 +175,23 @@ String String::trim() const
     return substring( firstNonSpace, lastNonSpace - firstNonSpace + 1 );
 }
 
+String String::trimZerosFromRight() const
```

Comment:
Wouldn't it make more sense to traverse the string from the right?

Reply:
It will. Fixed.